### PR TITLE
Add Dyninst

### PR DESCRIPTION
**Pull request by blue42u**
Dyninst only supports Glibc and not Musl libc. This PR includes a small patch to separate Alpine (`alpine`) from `linux` in ci_config.json, similar to the separation between `msys2` and `windows`.

### CODE REVIEW

**Review by neheb — APPROVED** [review]

**Comment by blue42u** [comment]
I see this has been approved, is this waiting on something before merge?

**Pull request merged**
